applications/CoSimulationApplication/tests/dem_fem_cable_net/ProjectParametersFEM.json

```diff
@@ -0,0 +1,62 @@
+{
+	"problem_data"             : {
+		"problem_name"    : "cableNetFEM",
+		"parallel_type"   : "OpenMP",
+		"start_time"      : 0.0,
+		"end_time"        : 1.5,
+		"echo_level"      : 1
+	},
+	"solver_settings"             : {
+		"solver_type"                     : "Dynamic",
+		"echo_level"                      : 1,
+		"analysis_type"                   : "non_linear",
+		"time_integration_method"         : "explicit",
+		"scheme_type"                     : "central_differences",
+		"time_step_prediction_level" 	  : 0,
+		"model_import_settings"   : {
+			"input_type"                  : "mdpa",
+			"input_filename"              : "dem_fem_cable_net/cableNetFEM"
+		},
+		"material_import_settings"    : {
+			"materials_filename"      : "dem_fem_cable_net/cableNetFEMStructuralMaterials.json"
+		},
+		"line_search"                         : false,
+		"convergence_criterion"               : "residual_criterion",
+		"displacement_relative_tolerance"     : 1e-9,
+		"displacement_absolute_tolerance"     : 1e-9,
+		"residual_relative_tolerance"         : 1e-9,
+		"residual_absolute_tolerance"         : 1e-9,
+		"max_iteration"                       : 400,
+		"problem_domain_sub_model_part_list"  : ["Parts_trusses"],
+		"processes_sub_model_part_list"       : ["DISPLACEMENT_dirichXYZ","RESULT_middleNode"],
```

Comment:
can you please remove `processes_sub_model_part_list` and `problem_domain_sub_model_part_list`, those should no longer be needed